frontend/packages/console-dynamic-plugin-sdk/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
+    "moduleResolution": "node",
+    "target": "es2016",
+    "declaration": true,
+    "outDir": "dist/lib",
+    "experimentalDecorators": true,
+    "sourceMap": true,
+    "noUnusedLocals": true,
+    "skipLibCheck": true
+  },
+  "files": ["src/webpack/ConsoleRemotePlugin.ts", "src/extensions/index.ts"]
```

Comment:
If we expose a `src/public/*` can we glob the public files ?

Reply:
we can use `include` instead of `files` in that case